src/Illuminate/Database/Console/PruneCommand.php

```diff
@@ -25,7 +25,8 @@ class PruneCommand extends Command
                                 {--model=* : Class names of the models to be pruned}
                                 {--except=* : Class names of the models to be excluded from pruning}
                                 {--chunk=1000 : The number of models to retrieve per chunk of models to be deleted}
-                                {--pretend : Display the number of prunable records found instead of deleting them}';
+                                {--pretend : Display the number of prunable records found instead of deleting them}
+                                {--delete-quietly : Delete the model quietly without firing the deleted event method}';
```

Comment:
I'd be tempted to make this simply `--quietly`.

Reply:
Thanks!